equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/crm14/Crm14DetailsModel.java

```diff
@@ -850,7 +851,7 @@ public boolean hasCrm15() {
         else if (StringUtils.isNotEmpty(this.getPartner_private_company()) && this.getPartner_private_company().equalsIgnoreCase("Yes"))
             return true;
         else
-            return this.getEmployers_crm15() > 0 || this.getPartner_employer_crm15() > 0;
+            return Integer.sum(this.getEmployers_crm15(),this.getPartner_employer_crm15()) > 0 || isCrm15Section();
     }
```

Comment:
this function does not cover all cases. It is missing 3 other questions. See PR for EMP-604. There is an update there